bowtie/_cli.py

```diff
@@ -1467,6 +1467,28 @@ def filter_benchmarks(
         console.Console().file.write(f"{file}\n")
 
 
+@subcommand
+@click.option(
+    "-r",
+    "--report",
+    "benchmark_report_files",
+    type=str,
+    multiple=True,
+    required=True,
+    help="Merge multiple benchmark reports into one.",
+)
+def merge_benchmark_reports(
```

Comment:
Didn't you say you thought we shouldn't add a CLI command because it didn't seem useful for anything else?

Reply:
But then how else are you suggesting that we use this ? Just have a function in _benchmarks.py which we can directly call using the python shell in the CI workflow ?

Reply:
Sure, `python -c 'from bowtie._benchmarks import combine; combine(sys.stdin)` is way better than 15 lines of untested python code written into a workflow!

Reply:
Got it, makes sense. Sorry for making things unnecessarily complicated.

Reply:
No no worries, I was terse!